module-os/board/linux/FreeRTOSConfig.h

```diff
@@ -52,39 +50,40 @@ extern "C"
 }
 #endif
 
-#define configUSE_PREEMPTION                    1
-#define configUSE_TICKLESS_IDLE                 0
-#define configCPU_CLOCK_HZ                      (SystemCoreClock)
-#define configTICK_RATE_HZ                      ((TickType_t)1000)
-#define configMAX_PRIORITIES                    5
-#define configMINIMAL_STACK_SIZE                ((unsigned short)128) //< Number of uint32_t's. So value of 128 will mean 512 bytes
-#define configMAX_TASK_NAME_LEN                 64
-#define configUSE_16_BIT_TICKS                  0
-#define configIDLE_SHOULD_YIELD                 1
-#define configUSE_TASK_NOTIFICATIONS            1
-#define configUSE_MUTEXES                       1
-#define configUSE_RECURSIVE_MUTEXES             1
-#define configUSE_COUNTING_SEMAPHORES           1
-#define configUSE_ALTERNATIVE_API               0 /* Deprecated! */
-#define configQUEUE_REGISTRY_SIZE               32
-#define configUSE_QUEUE_SETS                    1
-#define configUSE_TIME_SLICING                  1
-#define configUSE_NEWLIB_REENTRANT              0
+#define configUSE_PREEMPTION          1
+#define configUSE_TICKLESS_IDLE       0
+#define configCPU_CLOCK_HZ            (SystemCoreClock)
+#define configTICK_RATE_HZ            ((TickType_t)1000)
+#define configMAX_PRIORITIES          5
+#define configMINIMAL_STACK_SIZE      ((unsigned short)128) //< Number of uint32_t's. So value of 128 will mean 512 bytes
+#define configMAX_TASK_NAME_LEN       64
+#define configUSE_16_BIT_TICKS        0
+#define configIDLE_SHOULD_YIELD       1
+#define configUSE_TASK_NOTIFICATIONS  1
+#define configUSE_MUTEXES             1
+#define configUSE_RECURSIVE_MUTEXES   1
+#define configUSE_COUNTING_SEMAPHORES 1
+#define configUSE_ALTERNATIVE_API     0 /* Deprecated! */
+#define configQUEUE_REGISTRY_SIZE     32
+#define configUSE_QUEUE_SETS          1
+#define configUSE_TIME_SLICING        1
+
 #define configENABLE_BACKWARD_COMPATIBILITY     1
 #define configNUM_THREAD_LOCAL_STORAGE_POINTERS 5
 #define configUSE_APPLICATION_TASK_TAG          0
 
 /* Memory allocation related definitions. */
 #define configSUPPORT_STATIC_ALLOCATION  1
 #define configSUPPORT_DYNAMIC_ALLOCATION 1
-#define configTOTAL_HEAP_SIZE            ((size_t)(1024 * 324))
+#define configTOTAL_HEAP_SIZE ((size_t)(1024 * 512))
+
```

Comment:
do we have more space now on the ram somehow?

Reply:
https://github.com/mudita/MuditaOS/pull/3692#discussion_r840489814

Reply:
it's linux :facepalm: